src/com/theironyard/novauc/User.java

```diff
@@ -0,0 +1,72 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by Eric on 2/7/17.
+ */
+public class User {
+    String name;
+    double exp;
+    boolean online;
+    int onlineTime;
+    boolean streamUp;
+    char logo;
+
```

Comment:
Awesome job Eric.  Naming conventions were perfect.  Called methods in the Main class well. One very small thing Class names are generally nouns and are generally singular. So Credits should be Credit class.